src/Compilers/VisualBasic/Test/Semantic/Semantics/OptionalArgumentTests.vb

```diff
@@ -1087,12 +1087,6 @@ End Class
 BC30529: All parameters must be explicitly typed if any of them are explicitly typed.
             Optional f A String = "",
                      ~
-BC30812: Optional parameters must specify a default value.
```

Comment:
This compilation should be cloned, the original version should target VB14 language version, additional compilation should target VB15 version.